docs/twig.md

```diff
@@ -1,17 +0,0 @@
-# Twig
-
-This documents some of the fancy things you can do with Twig.
-
-## Extending Filters
-
-Add a `filters.metadata` file in the src directory....
-
-``` md
----
-slug: kalastatic-twig-filters/lib/filters/slug.js
----
```

Comment:
Yeah, all the stuff that's getting removed is the 1.0 wiki stuff. I probably didn't do it well. Maybe we can cherry pick the 1.0 wiki stuff into the 1.0 tag and then remove the commit that adds them here.